pkg/datapath/iptables/iptables.go

```diff
@@ -413,7 +413,7 @@ func (m *IptablesManager) Init() {
 		"ip6_tables", "ip6table_mangle", "ip6table_raw", "ip6table_filter"); err != nil {
 		if option.Config.EnableIPv6 {
 			log.WithError(err).Warning(
-				"IPv6 is enabled and ip6tables modules could not be initialized")
+				"IPv6 is enabled and ip6tables modules could not be initialized. Try loading ip6_tables, ip6table_mangle, ip6table_raw and ip6table_filter modules.")
```

Comment:
super nit: I would prefer that this be parenthetical.

```suggestion
				"IPv6 is enabled and ip6tables modules could not be initialized (try loading ip6_tables, ip6table_mangle, ip6table_raw and ip6table_filter modules)")
```

Reply:
I'd probably suggest using `log.WithField("hint", "Try loading ...")` instead, but I don't think it's really consequental. :)